buildbots.rst

```diff
@@ -193,45 +193,31 @@ or by making its parameters - such as a timeout - more robust.
 Custom builders
 ---------------
 
-When working on a long-lived feature branch, or on a bugfix branch which
-enables issue-specific debugging, you will probably want to test your
-work on one or several buildbots.  Since your work is hosted in a distinct
-repository, you can't trigger builds on the regular builders.  Instead,
-you have to use one of the `custom builders
+When working on a platform-specific issue, you may want to test your changes on
+the buildbot fleet rather than just on Travis and AppVeyor.  To do so, you can
+make use of the `custom builders
 <http://buildbot.python.org/all/waterfall?category=custom.stable&category=custom.unstable>`_.
+These builders track the ``buildbot-custom`` short-lived branch of the
+``python/cpython`` repository, which is only accessible to core developers.
```

Comment:
Thanks @zware. Sorry, I totally missed that line. It's fine as written or perhaps start the section with "Core developers have the option when working on a platform-specific issue...".